subscription/views.py

```diff
@@ -62,6 +62,7 @@
 ERROR_BUCKET_NOT_FOUND = "bucket not found for user"
 FREE_ACCESS_UNIT = 0
 PAID_ACCESS_UNIT = 1
+PAID_PAGE_DB_VALUE = 1
```

Comment:
Can you search the db by name instead of ID plz? i.e. AccessType = "Paid" instead of AccessTypeId = 1. Id is unreliable, and no one could easily understand what does it mean by PAID_PAGE_DB_VALUE = 1. Although the way it works is slightly different, you may refer to https://github.com/tair/api-python/blob/a97851747945867544c304d2cda483ab7cc9a86d/authorization/models.py#L53 and see how it is checked.

Reply:
its now searching using AccessType = "Paid" instead of the id.